_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and natural language processing (NLP) support tools for OpenSearch. It provides data analysts with the ability to:
+
+- Call OpenSearch indexes and manipulate using the opensearch-py-ml [DataFrame](https://opensearch-project.github.io/opensearch-py-ml/reference/dataframe.html) APIs. The opensearch-py-ml DataFrame wraps an OpenSearch index into a API similar to [pandas](https://pandas.pydata.org/), giving you the ability to process large amounts of data from OpenSearch inside a Jupyter Notebook.
+-  Upload NLP [SentenceTransformers](https://www.sbert.net/) models into OpenSearch using the [ML Commons plugin]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index/).
+- Train and tune SentenceTransformer models with ML Common and synthetic queries.
+
+## Prerequisites 
+
+To use `opensearch-py-ml`, install the [OpenSearch Python client]({{site.url}}{{site.baseurl}}/clients/python#setup). The Python client allows OpenSearch to use the Python syntax required to run Data Frames in `opensearch-py-ml`.
```

Comment:
Should "Data Frames" be "DataFrames" (it appears as one word above)?

Reply:
Yes, it should be DataFrames